### PR TITLE
added hello program

### PR DESCRIPTION
**Pull request by xiuhuiwang**
#2 finished.
Added hello program in Java.

### CODE REVIEW

File: helloworld.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class helloworld {
+    public static void main(String[] args) {
+        System.out.println("Hello World!");
```
Comment (tim61114) [inline helloworld.java]: I WANT ALL CAPS

**Review by tim61114 — CHANGES_REQUESTED** [review]
Please fix as requested

File: helloworld.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 public class helloworld {
     public static void main(String[] args) {
-        System.out.println("Hello World!");
+        System.out.println("ALL CAPS!");
```
Comment (tim61114) [inline helloworld.java]: I meant hello world in all caps 😢 
"HELLO WORLD!"

**Review by tim61114 — CHANGES_REQUESTED** [review]
Very close

File: helloworld.java (hunk 2 above)
Comment (xiuhuiwang) [reply helloworld.java]: I know:)

Comment (tim61114) [reply helloworld.java]: image

**Review by tim61114 — APPROVED** [review]
image

**Pull request merged**